src/Common/src/System/CommonRuntimeTypes.cs

```diff
@@ -31,8 +31,8 @@ internal static class CommonRuntimeTypes
         internal static Type IntPtr { get { return s_intPtr; } }
```

Comment:
Nit: While you are on it, you can reformat this to modern C#:

`internal static Type IntPtr { get; } = typeof(IntPtr);`